src/lang/de.json

```
@@ -134,9 +134,11 @@
 	"gui.toast.title.installed": "Mod installiert!",
 	"gui.toast.title.failed": "Fehler beim Installieren des Mods!",
 	"gui.toast.title.malformed": "Fehlerhafte Ordnerstruktur!",
+	"gui.toast.title.duped": "Duplizierter Ordner name!",
 	"gui.toast.desc.installed": "wurde installiert!",
 	"gui.toast.desc.malformed": "hat eine fehlerhafte Ordnerstruktur, falls du der Entwickler bist, solltest du dies beheben.",
 	"gui.toast.desc.failed": "Ein unbekannter Fehler ist aufgetaucht beim Installieren, die Schuld kann beim Autor liegen oder bei Viper selbst!",
+	"gui.toast.desc.duped": "hat multiple Ordner in sich mit dem selben Namen, wodurch ein duplizierter Ordner ensteht! Falls du der Entwickler bist solltest du dies beheben!",
```

```suggestion
	"gui.toast.desc.duped": "hat mehrere Ordner in sich mit dem selben Namen, wodurch ein duplizierter Ordner ensteht! Falls du der Entwickler bist solltest du dies beheben!",
```

"mehrere" sounds better than "multiple", the latter isn't even a proper German word ^^

Source: I speak German :P

---

"multiple" Indeed isn't german but it is still being used in the german language.
We often refer to it as "Eingedeutschtes Wort", which basically just means this Word is being used in german sentences even thou it is not german. (Or we change up the word from another language and add it to german)

---

I don't speak a word of German, so I don't know which German to trust here...

---

Both are okay but I have to say that @GeckoEidechse solution sounds better and would be easier to understand for users.